KPI-team1-frontend/src/App.tsx

```diff
@@ -3,14 +3,30 @@ import Sidebar from "./components/Sidebar";
 import { supabase } from "./supabase";
 import { useEffect, useState } from "react";
 import { User } from "./model/user";
+import { HiOutlineMagnifyingGlass } from "react-icons/hi2";
+import { PiBell } from "react-icons/pi";
+import { Circles } from "./model/circle";
 
 const initialUser: User = {
   id: "",
   email: "",
 };
 
+// const initialCircles: Circles = {
```

Comment:
It is good practice IMHO to not have code that is commented out unless we have a very good reason - usually it gets stale very quickly.